mod/manifest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+    use Friendica\Core\Config;
+
+    function manifest_content(App $a) {
+
+		$tpl = get_markup_template('manifest.tpl');
+
+		header('Content-type: application/manifest+json');
+
+		$touch_icon = Config::get('system', 'touch_icon', 'images/friendica-128.png');
+		if ($touch_icon == '') {
```

Comment:
Hmm, you're right, and there's no visual cue in the admin that the setting is unset or just blank.